src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -507,17 +509,17 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, err, tim
               ! compute the turbulent scales in order to compute the bulk Richardson number
               call cvmix_kpp_compute_turbulent_scales( &
                    sigma_coord = config_cvmix_kpp_surface_layer_extent, &
-                   OBL_depth = OBLDepths(1:maxLevelCell(iCell)), &
-                   surf_buoy_force = interfaceForcings(1:maxLevelCell(iCell)), &
+                   OBL_depth = OBLDepths(minLevelCell(iCell):maxLevelCell(iCell)), &
+                   surf_buoy_force = interfaceForcings(minLevelCell(iCell):maxLevelCell(iCell)), &
                    surf_fric_vel = cvmix_variables % SurfaceFriction, &
-                   w_s = turbulentScalarVelocityScale(1:maxLevelCell(iCell)))
+                   w_s = turbulentScalarVelocityScale(minLevelCell(iCell):maxLevelCell(iCell)))
 
               ! averaging over a surface layer assuming that BLdepth is cell bottom
               ! Build deltaVelocitySquared
               do i = 1, nEdgesOnCell(iCell)
                  iEdge = edgesOnCell(i, iCell)
 
-                 deltaVelocitySquared(1:minLevelCell(iCell)) = 0.0_RKIND
+                 deltaVelocitySquared(minLevelCell(iCell):minLevelCell(iCell)) = 0.0_RKIND
```

Comment:
It's not clear to me why this should be changed.

Reply:
I can change it back. I don't think we ever use values `k<minLevelCell(iCell)`

Reply:
good point.  Could we instead change to
```
deltaVelocitySquared(minLevelCell(iCell)) = 0.0_RKIND
```
I don't see the need for array indexing.

Reply:
Oh, I totally missed the fact that the start and end indices were the same. Of course.

Reply:
Actually, I think the most likely thing going on here is that the original code is from phase 1 and you're modifying it again in phase 2, but should probably leave it as it was in phase 1.

Reply:
I would assume anything that already has a `minLevel*` has already been handled.

Reply:
Good point. I'll leave it as it was.